app/src/main/java/org/listenbrainz/android/ui/screens/artist/ArtistScreen.kt

```diff
@@ -1067,56 +1067,6 @@ private fun LbRadioButton(
     }
 }
 
-@Composable
-fun SvgWithWebView(svgContent: String, width: Dp, height: Dp) {
```

Comment:
Lets not remove this function but rather document its behavior since it has no usage as of now.